.github/workflows/build_new_version.yml

```diff
@@ -33,6 +35,8 @@ jobs:
         with:
           images: |
             ghcr.io/borgmatic-collective/borgmatic
+          flavor: |
+            latest=false
```

Comment:
Do we want to set this to false if it’s the base?

Reply:
I cannot decide on that but here are the two different scenarios explained.

Current implementation:
Every time a tag is pushed the base image will get assigned the latest tag since the auto mode is triggered by `type=semver,pattern=...`.

Now that I think about it the conditional latest(-ntfy/msmtp) tagging doesn't make sense since it would only apply when it's pushed to the default branch. Afaik tags and branches are not directly related and tagging just cares about the commits. So these would never trigger as seen with previous runs for this specific workflow. To be honest I really don't get the need for the two different workflow runs but I'd propose to change the ntfy/msmtp-pushes to be changed to `type=raw,value=latest-ntfy/msmtp,enable=true` (enable could be omitted but explicitly stating it makes it easier to understand) and remove the flavor changes and `type=raw,...` from the base configuration.